docs/metrics.md

```diff
@@ -1,3 +1,3 @@
 # Metrics
 
-To find more about the Eventing metrics API please visit the related [page](https://knative.dev/docs/admin/collecting-metrics/eventing-metrics/metrics/) in Knative docs. For collecting and visualizing Eventing metrics check the [monitoring sandbox project](https://github.com/knative-sandbox/monitoring) or for an alternative approach check this [page](https://knative.dev/docs/admin/collecting-metrics/#set-up-the-collector) in Knative docs.
+To find more about the Eventing metrics API please visit the [metrics page](https://knative.dev/docs/admin/collecting-metrics/eventing-metrics/metrics/) in Knative docs. For collecting and visualizing Eventing metrics check the [monitoring sandbox project](https://github.com/knative-sandbox/monitoring) or for an alternative approach check the [collecting metrics page](https://knative.dev/docs/admin/collecting-metrics/#set-up-the-collector) in Knative docs.
```

Comment:
Should we link to this page in `metrics page` https://knative.dev/docs/eventing/observability/metrics/eventing-metrics/ ?